torch/ao/quantization/observer.py

```diff
@@ -628,6 +637,11 @@ def __init__(
         factory_kwargs=None,
         eps=torch.finfo(torch.float32).eps,
     ) -> None:
+        if qscheme != torch.per_channel_symmetric and qscheme != torch.per_channel_affine:
```

Comment:
there is one qscheme missing here, but can be fixed if we use `is_per_channel` to check this

Reply:
Changed.